lib/process-services-cloud/src/lib/process/process-list/components/process-list-cloud.component.ts

```diff
@@ -156,6 +157,7 @@ export class ProcessListCloudComponent extends DataTableSchema implements OnChan
     selectedInstances: any[];
     isLoading = true;
     rows: any[] = [];
+    sortInput: any[];
```

Comment:
the `Input` suffix makes no sense, can you just call it `sorting` like the DataTable does that? in that case it will be intuitive for developers